invalid error tag in diagnostic control option: extern_entity_treated_as_static

Hi, I was running the [example](https://github.com/neworderofjamie/multi-area-model/blob/6b3d20a9e47ec054ec2299f323f4191dbb942cf2/run_example_downscaled.py), and I've got this error
```
make: Entering directory '/home/lz/procedural_paper/models/multi-area-model/multi_area_model_CODE'
Command-line error: invalid error tag in diagnostic control option: extern_entity_treated_as_static
1 catastrophic error detected in this compilation.
Compilation terminated.
Command-line error: invalid error tag in diagnostic control option: extern_entity_treated_as_static
1 catastrophic error detected in this compilation.
Compilation terminated.
Makefile:22: recipe for target 'synapseUpdate.o' failed
Makefile:22: recipe for target 'customUpdate.o' failed
make: *** [synapseUpdate.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [customUpdate.o] Error 1
Command-line error: invalid error tag in diagnostic control option: extern_entity_treated_as_static
1 catastrophic error detected in this compilation.
Compilation terminated.
Command-line error: invalid error tag in diagnostic control option: extern_entity_treated_as_static
1 catastrophic error detected in this compilation.
Compilation terminated.
Makefile:22: recipe for target 'neuronUpdate.o' failed
Makefile:22: recipe for target 'init.o' failed
make: *** [neuronUpdate.o] Error 1
make: *** [init.o] Error 1
Command-line error: invalid error tag in diagnostic control option: extern_entity_treated_as_static
1 catastrophic error detected in this compilation.
Compilation terminated.
make: *** [runner.o] Error 1
Makefile:22: recipe for target 'runner.o' failed
make: Leaving directory '/home/lz/procedural_paper/models/multi-area-model/multi_area_model_CODE'
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/lz/pycharm/plugins/python-ce/helpers/pydev/_pydev_bundle/pydev_umd.py", line 197, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "/home/lz/pycharm/plugins/python-ce/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/lz/procedural_paper/models/multi-area-model/run_example_downscaled.py", line 40, in <module>
    M.simulation.simulate()
  File "/home/lz/procedural_paper/models/multi-area-model/multiarea_model/simulation.py", line 308, in simulate
    self.model.build()
  File "/home/lz/procedural_paper/genn/pygenn/genn_model.py", line 579, in build
    check_call(["make", "-j", str(cpu_count(logical=False)), "-C", output_path])
  File "/home/lz/anaconda3/envs/pcl/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-j', '36', '-C', './multi_area_model_CODE']' returned non-zero exit status 2.
```
The OS is Ubuntu with CUDA9.0.

--- a/src/genn/backends/cuda/backend.cc
+++ b/src/genn/backends/cuda/backend.cc
@@ -1677,7 +1677,7 @@ std::string Backend::getNVCCFlags() const
 #ifndef _WIN32
     nvccFlags += " -std=c++11 --compiler-options \"-fPIC -Wno-return-type-c-linkage\"";
 #endif
-    if(m_RuntimeVersion >= 9000) {
+    if(m_RuntimeVersion >= 9020) {
         nvccFlags += " -Xcudafe \"--diag_suppress=extern_entity_treated_as_static\"";
     }
 

